src/environmentd/src/http/webhook.rs

```diff
@@ -131,6 +132,11 @@ async fn append_webhook(
     }
     .await?;
 
+    // These must happen before validation as we do not know if validation or
+    // packing will succeed and appending will begin
+    tx.increment_messages_received(1);
+    tx.increment_bytes_received(u64::cast_from(body.len()));
```

Comment:
What do you think about including the size of the headers here? Personally I'm 50/50 on it, but figured I would call it out

Reply:
I think its fine to just have the values; we don't report extra info like this for other sources!